appveyor.yml

```diff
@@ -1,31 +1,5 @@
 version: "{branch}-{build}"
 build: off
-cache:
-  - 'C:\\tmp'
-environment:
-  matrix:
-    - PYTHON: "C:\\Python36"
-      TOXENV: "py36,py36-six"
```

Comment:
What do you think? If we keep the `six` test on linux / travis for now, should we also keep those here? IMO the travis ones are probably enough.

Reply:
Yeah, basically I want to get rid of appveyor :)

Reply:
Maybe we can also get rid of non six tests, we'll see in #965